pydash/arrays.py

```diff
@@ -1424,7 +1466,7 @@ def take_while(array, callback=None):
     return array[:n]
 
 
-def union(*arrays):
+def union(arrays):
```

Comment:
Actually, take a look at `intersection`, `intersection_by`, and `intersection_with`. We should model the function signatures the same for the `union*` functions. There's also special handling for passing in the callback as a positional argument that can be used in `union_by` and `union_with`.